tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/MinorDottySuite.scala

```diff
@@ -924,4 +924,81 @@ class MinorDottySuite extends BaseDottySuite {
       )
     )
   }
+
+  // infix modifier - can be appplied only to type and methods
+  test("infix-modifier") {
+
```

Comment:
it should work okay but I would just add a test for infix extension method

Reply:
Done!